### PR TITLE
Fixe IOS issues

### PR DESCRIPTION
**Pull request by kefahB**
Hi @triniwiz,

I've fixed the issues #134 and #137 and partially the #135, this fixe reduce the number of call but not fixe it definitely

### CODE REVIEW

**Comment by kefahB** [comment]
@triniwiz thanks .. please don not forged to take care of #133 please

File: packages/nativescript-stripe/standard/index.ios.ts [hunk 1]
```diff
@@ -211,11 +211,15 @@ class StripePaymentDelegate extends NSObject implements STPPaymentContextDelegat
 	paymentContextDidCreatePaymentResultCompletion(paymentContext: STPPaymentContext, paymentResult: STPPaymentResult, completion: (p1: STPPaymentStatus, p2: NSError) => void): void {
 		StripeStandardConfig.shared.backendAPI
 			.capturePayment(paymentResult.paymentMethod.stripeId, paymentContext.paymentAmount, createShippingMethod(paymentContext), createAddress(paymentContext.shippingAddress))
-			.then((value) => {
-				completion(STPPaymentStatus.Success, null);
+			.then((value: any) => {
+                if(!value._native.lastPaymentError || value._native.lastPaymentError == "undefined") {
```
Comment (justinkurtz) [inline packages/nativescript-stripe/standard/index.ios.ts:215]: @kefahB @triniwiz I'm having trouble with this change. Every payment fails with the error originating from this line:
`Cannot read property 'lastPaymentError' of undefined`

In my case, the variable "value" is the result of the promise returned from capturePayment defined in my own code. This is referring to a _native property which my promise doesn't return. Seems like this line of code is supposed to call something else internal to the Stripe pod to get the lastPaymentError?

**Review by justinkurtz — COMMENTED** [review]
@kefahB @triniwiz Having an issue with the changes to fix the payment result handler. I think there might be a mistake but I'm happy help.

**Comment by kefahB** [comment]
Hi @justinkurtz 

Strange! before those changes the payment always resolve true even if it is in reality not !

Can you share how do you proceed to capturePayment the payment please ? do you use standard or sheet implementation  ?

**Comment by justinkurtz** [comment]
I think I'm using the standard implementation. I've taken this example and modified it:
`apps/demo/src/plugin-demos/nativescript-stripe-src/std-view/stripe.service.ts`

I see now that the capturePayment method has been changed since I originally adapted it to my project. Previously (in the commented out code) it just made an HTTP post and returned that. Seems as though I need to adapt the new code in capturePayment from that example.

**Comment by justinkurtz** [comment]
I wonder if we can add a null safe navigation operator to support use cases that don't immediately confirm payment intents or use 3DS. Changing this line to the below would give more flexibility and make it backwards compatible.

`if(!value._native?.lastPaymentError || value._native?.lastPaymentError == "undefined") {`

**Comment by kefahB** [comment]
Effectively it is a good idea .. @triniwiz what do you thik ?

**Comment by triniwiz** [comment]
Sounds good 👌

**Pull request merged**